.gitignore

```diff
@@ -1,17 +1,28 @@
-.gradle/
-eclipse/
-out/
-Jabba.iml
-Jabba.ipr
-Jabba.iws
-*.bat
-
-######################################
-## Remove MAC OSX generated garbage
-######################################
-
-*/.DS_Store
```

Comment:
These macOS generated garbage .gitignores are useful for me; on Mac so I end up just adding them back in

Reply:
Please open an issue or even PR on example mod! Good to know!